examples/fastapi/tests.py

```diff
@@ -0,0 +1,39 @@
+# mypy: no-disallow-untyped-decorators
```

Comment:
Add `# pylint: disable=E0611` here can resolve lint error with CI

Reply:
yes, it is worked, thinks :)